plugin/CactbotEventSource/FFXIVProcessCn.cs

```diff
@@ -7,80 +7,82 @@
 
 namespace Cactbot {
   public class FFXIVProcessCn : FFXIVProcess {
-    // Last updated for FFXIV 5.4
-    //
-    // Latest CN version can be found at:
-    // http://ff.sdo.com/web8/index.html#/patchnote
+        // Last updated for FFXIV 6.0
+        //
+        // Latest CN version can be found at:
+        // http://ff.sdo.com/web8/index.html#/patchnote
 
-    [StructLayout(LayoutKind.Explicit)]
-    public unsafe struct EntityMemory {
-      public static int Size => Marshal.SizeOf(typeof(EntityMemory));
+        [StructLayout(LayoutKind.Explicit)]
+        public unsafe struct EntityMemory
+        {
```

Comment:
Thank you for the update.  Sorry for the small comment, but is it possible to not reformat this file by adding extra spaces or putting `{` on a different line?